tests/test_gh_workflows.py

```diff
@@ -26,6 +26,6 @@ def test_matrix_to_lock_version_defaults(self):
         lock_workflow = yaml.safe_load(lock_workflow_file.read_text())
         lock_versions = lock_workflow[True]['workflow_dispatch']['inputs']['branches']['default'].split(',')
 
-        matrix_versions = get_stack_versions()
+        matrix_versions = get_stack_versions(drop_patch=True)
```

Comment:
If we drop the patch here, do we need to shrink the list as done in line 31?

Reply:
that will drop the last entry of the matrix altogether (not just the patch) 👍